src/labeler.ts

```diff
@@ -38,24 +47,38 @@ export async function run() {
       configPath
     );
 
-    const labels: string[] = [];
+    const labelsToAdd: string[] = [];
     const labelsToRemove: string[] = [];
+    const preexistingLabels = pullRequest.labels
+      .map((l) => l.name)
```

Comment:
A nitpick: i would rather omit the parenthesis around the single arguments. They are verbose and do not correspond to the style of the existing code.